cli/injector/main.go

```diff
@@ -308,9 +359,58 @@ func inject(kind string, sendToMetrics bool) bool {
 	return !errOnInject
 }
 
+// reinject reinitialize conf, clean and inject all the disruptions
+func reinject(pod *v1.Pod, cmdName string) error {
+	var err error
+
+	// Clean all injections to reinject on an empty slate
+	if ok := clean(cmdName, true, true); !ok {
```

Comment:
Why do we need to clean? Do all disruption kinds need to clean? I would think we could just edit the cgroup files for the new container id, what complications am I overlooking?